algorithms/unit_tests/TestStdAlgorithmsUniqueCopy.cpp

```diff
@@ -322,6 +322,9 @@ void run_all_scenarios() {
 }
 
 TEST(std_algorithms_mod_seq_ops, unique_copy) {
+#if defined(KOKKOS_ENABLE_SYCL) && !defined(KOKKOS_ARCH_INTEL_GPU)
```

Comment:
Should this be `&& !defined(KOKKOS_ENABLE_CUDA)`?

Reply:
No, you can't enable multiple backends anyway. I could replace `!defined(KOKKOS_ARCH_INTEL_GPU)` with something like `(defined(KOKKOS_ARCH_VEGA) || defined(KOKKOS_ARCH_AMPERE) || ...)` but I'm not convinced that it's worth it.
You need to provide an architecture flag when using `SYCL+Cuda` but otherwise, you don't. Hence, this check also restricts the tests run on, say `Intel GPU`s, if no architecture was specified (but then we don't enable some optimizations).